### PR TITLE
Capstone

### PR DESCRIPTION
**Pull request by aradradev**
**Capstone project for the first module**
- Added a workflow in the project
- Added a professional Readme file
- Added all things that I learned during the module

### CODE REVIEW

**Review by koachcode — COMMENTED** [review]
Hi @aradradev, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not meet the README requirements for  the capstone, no video presentation, no Live link, and more. Please go through the requirements and implement the necessary updates to your project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+If you like this project, please give me a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="Cindy">https://www.behance.net/gallery/29845175/CC-Global-Summit-2015</a>
+
+
+I would like to thank Microverse for this exercise and also pass my gratitude to Cindy Shin for designing the template and make it available for use. 
+
```
Comment (koachcode) [inline README.md]: - Please this should be the link to the author's Behance handle.  as in the requirement
        -  The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+- [Live Demo Link](https://aradradev.github.io/first-capstone-project/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Communicate technical concepts to non-technical people Loom video
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
```
Comment (koachcode) [inline README.md]: - Please add your presentation video of the project showing the features of desktop and mobile versions. The maximum length of the video is 5 minutes.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        <img src="images/icon-close.svg" alt="icon-close" id="close-btn">
+      </div>
+      <nav class="navbar mobile">
+        <ul>
+          <li><a href="#">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Contact</a></li>
+          <li><a href="#">Blog</a></li>
+          <li><a href="#">Careers</a></li>
+        </ul>
```
Comment (koachcode) [inline index.html:39]: - From the project requirements both the Home page and About page should have the same Navbar and Footer, but seems your Navbars are different is different, please make them the same.  Both should look like the home page Navbar

##### Your home page nav bar 

image

##### About page nav bar
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+    </ul>
+  </section>
+
+  <footer class="footer">
+    <ul>
+      <li><img src="images/icon-facebook.svg" alt="icon-facebook"></li>
+      <li><img src="images/icon-instagram.svg" alt="icon-instagram"></li>
+      <li><img src="images/icon-twitter.svg" alt="icon-twitter"></li>
+      <li><img src="images/icon-youtube.svg" alt="icon-youtube"></li>
+    </ul>
```
Comment (koachcode) [inline index.html:155]: - from the requirements the Footer should have a logo aligned to the left and text aligned to the right. Please  Folow the structure of the original design. 

##### Your Footer 

image

##### Original Design Footer

image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+
+    <section class="about">
+      <div class="about-section">
+        <h3>Why choose Easybank?</h3>
+        <p>We leverage Open Banking to turn your bank account into your financial hub. Control your finances like never before.</p>
+      </div>
+      <div class="cards">
+        <div class="card">
+          <img src="images/icon-online.svg" alt="icon-online">
+          <h3>Online Banking</h3>
```
Comment (koachcode) [inline index.html]: - From the requirements, Home page should contain a middle section with a column that includes a section header, a divider,` five individual articles, each with an image, title, and description, and a line of text at the bottom.` Please as at present you have just four, please make it five. 


##### Your design

image

##### Main design 
image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+          <p>Our modern web and mobile applications allow you to keep track of your finances wherever you are in the world.</p>
+        </div>
+        <div class="card">
+          <img src="images/icon-budgeting.svg" alt="icon-budgeting">
+          <h3>Simple Budgeting</h3>
+          <p>See exactly where your money goes each month. Receive notifications when you’re close to hitting your limits.</p>
+        </div>
+        <div class="card">
```
Comment (koachcode) [inline index.html:117]: - On the mobile screen, you should have The middle section individual articles have their elements organized in an horizontal row, with the image at the left, the title in the center, and the description at the right.


##### Your design 
image

##### Main Design

image

File: about.html [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+          <div class="parag">
+            <p>Our app is designed to be easy to use, so you can take control of your money with just a few taps on your smartphone. Whether you need to check your balance, transfer funds, or pay bills, EasyBank has got you covered.</p>
+          </div>
+          <div class="parag">
+            <p>We also offer a wide range of financial planning tools to help you save for your future goals, whether that's buying a house, starting a business, or retiring comfortably.</p>
+          </div>
+          <div class="parag">
+            <p>So why wait? Download the EasyBank app today and start enjoying the benefits of banking made easy.</p>
+          </div>
+          <div class="date">
```
Comment (koachcode) [inline about.html]: - Please you are to follow the layout of the original design. Below are the requirements.
    1. It contains a top section with a central column that includes a page title, a text block with the page description, and a line with contact information or any other prompt under it.
   1. It contains a middle section with a central column that includes a section header, a divider, a paragraph of text, and an image or large logo.
   1. It contains a bottom section with a central column that includes a section header, a divider, a line of text, and two images with text centered over them.  


##### Your Design 

image


##### Original Design 
image

File: index.html [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+        <div class="image-desk">
+          <img class="hidden" src="images/image-mockups.png" alt="image-mockups" width="100%">
+        </div>
+      </div>
+      <div class="image">
+        <img class="mobile" src="images/image-mockups.png" alt="image-mockups" width="100%">
+      </div>
+    </section>
+
+    <section class="about">
```
Comment (koachcode) [inline index.html:102]: - Please use an HTML `article tag` in other to have a semantic HTML, and this section is an article section so it has an enclosed article tag.

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @aradradev , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!



## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please go through the requirements as it is paramount that you have the same layout as the original design.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 9]
```diff
@@ -0,0 +1,207 @@
...
+If you like this project, please give me a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="Cindy">https://www.behance.net/adagio07</a>
+
+
+I would like to thank Microverse for this exercise and also pass my gratitude to Cindy Shin for designing the template and make it available for use.
```
Comment (Olamarx) [inline README.md:181]: Good job adding link of the designer. 👍🏼

File: index.html (hunk 6 above)
Comment (Olamarx) [reply index.html:117]: > * On the mobile screen, you should have The middle section individual articles have their elements organized in an horizontal row, with the image at the left, the title in the center, and the description at the right.
> 
> ##### Your design
> image
> 
> ##### Main Design
> image

- [ ] While you have done a good job, I will implore you to make the text smaller and even give the tag a definite size that will make the cards the same across the board.

### Your project
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @aradradev, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Linters passed ✔️
- Readme file professional ✔️
- Requested changes done ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 10]
```diff
@@ -387,6 +390,82 @@ hr {
   text-align: center;
 }
```
Comment (zieeco) [inline style.css:392]: ### CARDS

**[OPTIONAL]**

- Great job with your changes, however, I think these sections should look the same as the vertical layout in your desktop version, while you've added a breakpoint that makes it look like the main design, kindly make it the same on all other desktop screen sizes.


*Update this as seen in a bigger screen*
Screenshot from 2023-05-22 17-54-39

*to this also in bigger screen*
Screenshot from 2023-05-22 17-54-26

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @Gmanlove, 




Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
*Check the comment under the review*

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**